dotnet/src/Extensions/Planning.SequentialPlanner/SKContextSequentialPlannerExtensions.cs

```diff
@@ -6,15 +6,13 @@
 using System.Threading.Tasks;
 using Microsoft.Extensions.Logging;
 using Microsoft.SemanticKernel.Memory;
-using Microsoft.SemanticKernel.Planning.Planners;
+using Microsoft.SemanticKernel.Planning.Sequential;
 using Microsoft.SemanticKernel.SkillDefinition;
 
-#pragma warning disable IDE0130
```

Comment:
Please leave these in and add elsewhere as appropriate.

Reply:
what are they for?

Reply:
I don't see any build warning after removing them

Reply:
added everywhere I could find it - didn't know about the feature, nice one